Gordon360/Models/CCT/Participant.cs

```diff
@@ -0,0 +1,41 @@
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
+#nullable disable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Gordon360.Models.CCT
+{
+    [Table("Participant", Schema = "RecIM")]
+    public partial class Participant
+    {
+        public Participant()
+        {
+            MatchParticipant = new HashSet<MatchParticipant>();
+            ParticipantActivity = new HashSet<ParticipantActivity>();
+            ParticipantNotification = new HashSet<ParticipantNotification>();
+            ParticipantStatusHistory = new HashSet<ParticipantStatusHistory>();
+            ParticipantTeam = new HashSet<ParticipantTeam>();
+        }
+
+        [Key]
+        [StringLength(50)]
+        [Unicode(false)]
+        public string Username { get; set; }
+        public bool IsAdmin { get; set; }
+        public int ID { get; set; }
```

Comment:
ID was a separate number completely disconnected from the gordonID. This was a half baked implementation in the event we want to handle _private_ students which would allow us to have them completely anonymous.